cdk/src/stacks/living-documentation.ts

```diff
@@ -37,6 +35,20 @@ export class LivingDocumentationStack extends NestedStack {
 
     (dashboardGeneratorFunction.node.defaultChild as lambda.CfnFunction).overrideLogicalId('DashboardGeneratorFunction');
 
+    const superwerkerDashboard = new Dashboard(this, 'SuperwerkerDashboard', {
+      dashboardName: 'Superwerker-LivingDocumentation',
+    });
+
+    superwerkerDashboard.addWidgets(
+      new CustomWidget({
+        title: '',
+        width: 20,
```

Comment:
Is there an option to use full width?

Reply:
There are 24 columns. So the maximum is 24. Let's try this.